### DISCUSSION TITLE
Welcome to weewx-belchertown Discussions!

### DISCUSSION THREAD

**Discussion by poblabs** [poblabs/weewx-belchertown#744] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DaleReid** [comment]
Just like weather and presenting it to the world.

**Comment by DaleReid** [comment]
I may have not seen the answer in the wiki, but here is a problem I'm having.
I set up weewx and added Belchertown when I learned of it.  I run a RainWise IP-100, and publish it to my GoDaddy server for viewing from the internet.

I got a Raspberry Pi 400, set it up with the recommended software, updated everything.  I installed WeeWx 4.8.0 and since I'm not feeding WeeWx with a station yet, set up the Simulator option.  That runs well and had been running and graphing with WeeWx stuff for weeks.

I added Belchertown skin.  I copied the graphs.conf.example into graphs.conf.  I have the four stanzas with the graphs for the nice charting that I chose Belchertown for.  

With the original system publishing to the internet webserver, I can go to that skin and see the Belchertown display, the Windy radar, and then below the four (now five with wind rose added), no problem.

I go to my new setup and go to the /var/www/html/weewx/belchertown and find the index.html and when I click on it, the browser opens and I see the header,  My Station Weather Conditons,with the temp in big numbers with feels like, high and low, and then the Barometer panel with dwpoint, humidity and rain in there, and the Windy graphic to the right of that.
Next grouping has the wind direction and sped with speed and gust,and then Sun and Moon panel.

DIrectly under that, the wheels seem to come off, with the almanac info expanded, and the Weather Record Snapshots just above the end of the page.  I can't show you a picture of  it since I don't have this on the web.  

I go to my functioning Rasp Pi 4 which has been running for a long time, and I get the same thing.    There is no visible graphing panels below,like show up on the published web page.  So the same Rasp Pi running this seems not to have a 'local' display of the info so I can verify and tweak it as I might.

Is the ONLY place that one can see those nice graphs is on an internet published web page?  Is there nothing locally on either of the Raspberries (the 4 or the 400?)  I was assuming that the Belchertown created a local set of graphs and those got sent to my web page, but I cannot find where they might be, or if they even exit.

Can someone help me understand how they can be on my web page (fed with a live station) and yet the new setup going through testing and configuration and having no site to publish to, cannot be seen?  Thanks for any guidance or pointers in the wiki that may help me.
Dale
